### PR TITLE
Cli update

### PR DESCRIPTION
**Pull request by gauntface**
Implements comments from: #213

@marco-c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,24 @@ registration.pushManager.subscribe({
 });
 ```
 
+## Command Line
+
+You can install `web-push` globally and use it for send notifications and / or
```
Comment (marco-c) [inline README.md]: Nit: sending

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,3 @@
 coverage/
 node_modules/
-wires
```
Comment (marco-c) [inline .gitignore]: Unneeded change?

Comment (gauntface) [inline .gitignore]: wires isn't generated by anything in this project.

We used to download and name a file wires for selenium, then selenium looked for 'geckodriver' which we did but saved somewhere else and now we just use the geckodriver module.

Comment (marco-c) [inline .gitignore]: Oh, right. I still have the `wires` executable in my folder 😄

**Pull request merged**